server/controllers/clubProfileController.js

```diff
@@ -1,9 +1,26 @@
 const db = require('../models/')
 
+function shuffle(sourceArray) {
+    for (var i = 0; i < sourceArray.length - 1; i++) {
+        var j = i + Math.floor(Math.random() * (sourceArray.length - i));
```

Comment:
what happens when this exceeds sourceArray.length? maybe consider simply having `j = randomNumber` rather than `j = i + randomNumber`?